examples/src/main/java/org/apache/spark/examples/ml/JavaKMeansExample.java

```diff
@@ -51,9 +52,17 @@ public static void main(String[] args) {
     KMeans kmeans = new KMeans().setK(2).setSeed(1L);
     KMeansModel model = kmeans.fit(dataset);
 
-    // Evaluate clustering by computing Within Set Sum of Squared Errors.
-    double WSSSE = model.computeCost(dataset);
-    System.out.println("Within Set Sum of Squared Errors = " + WSSSE);
+    // Make predictions
+    Dataset<Row> predictions = model.transform(dataset);
+
+    // Evaluate clustering by computing Silhouette score
+    ClusteringEvaluator evaluator = new ClusteringEvaluator()
+      .setFeaturesCol("features")
+      .setPredictionCol("prediction")
```

Comment:
We use default values here, so it's not necessary to set them explicitly. We should keep examples as simple as possible. Thanks.